src/main/resources/application.yaml

```diff
@@ -2,12 +2,15 @@ spring:
   jpa:
     show-sql: true
     hibernate:
-      ddl-auto: update
+      ddl-auto: validate
   datasource:
     driver-class-name: org.postgresql.Driver
-    url: ${SPRING_DATASOURCE_URL}
-    password: ${SPRING_DATASOURSE_PASSWORD}
-    username: ${SPRING_DATASOURSE_USERNAME}
+#    url: ${SPRING_DATASOURCE_URL}
+#    password: ${SPRING_DATASOURSE_PASSWORD}
+#    username: ${SPRING_DATASOURSE_USERNAME}
+    url: jdbc:postgresql://localhost:5432/internship
```

Comment:
Please, revert these changes to previous state.

Reply:
done